backend/src/main/java/io/debezium/configserver/model/ConnectorDefinition.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.model;
+
+import java.util.List;
+
+public class ConnectorDefinition extends ConnectorType {
```

Comment:
I'm sorry @mdrillin. This is still the wrong direction.
A `ConnectorType` should be the FULL thing including properties. As it was before.

A `SomeOtherThing`, for example `ConnectorDescriptor` is without properties.

Reply:
In my mind 'ConnectorType' is a smaller set of info than a 'ConnectorDescriptor'.  But no problem, if you guys want it the opposite way - I can change it.